.circleci/config.yml

```diff
@@ -18,10 +18,21 @@ jobs:
           command: sudo apt-get install -y npm
       - run:
           name: Install Dependencies
-          command: npm install --save-dev -y jest
+          command: |
```

Comment:
Added all dependencies trying to fix precision issue. It didnt, but I fixed the code though.